src/main/scala/org/ergoplatform/modifiers/mempool/ErgoTransaction.scala

```diff
@@ -100,6 +103,7 @@ case class ErgoTransaction(override val inputs: IndexedSeq[Input],
     validateStateless().result.toTry
   }
 
+  val verifyScriptMetric = MetricDesc[InputMetricData]("verifyScript")
```

Comment:
Add ScalaDoc for added vals and defs here and other classes.

Reply:
done